tox.ini

```diff
@@ -1,11 +1,11 @@
 [tox]
 envlist =
-    django18-py{27,34,35}
-    django18-py{27,34,35}-drf33
-    django19-py{27,34,35}
-    django19-py{27,34,35}-drf33
-    django110-py{27,34,35}
-    django110-py{27,34,35}-drf33
+    django18-py{27,35}
```

Comment:
I see what you did there... we might have to state this somewhere in the documentation (README and ???) that Python3.4 is not supported anymore.